src/ParameterDistributions.jl

```diff
@@ -369,7 +369,8 @@ function ParameterDistribution(param_dist_dict::Union{Dict, AbstractVector})
 
     # flatten the structure
     distribution = getindex.(param_dist_dict_array, "distribution")
-    flat_constraint = cat(getindex.(param_dist_dict_array, "constraint")..., dims = 1)
+    flat_constraint = reduce(vcat, getindex.(param_dist_dict_array, "constraint"))
+    flat_constraint = isa(flat_constraint, Vector) ? flat_constraint : [flat_constraint]
```

Comment:
Replace `cat(cons..., dims=1)` by `reduce(vcat, cons)`. This is generally faster and less allocating. In the case where `cons` is a 1-vector, the `reduce` operation gives a `Constraint` instead of a `Vector{Constraint}`, therefore L373 is needed to ensure we end up with a vector.